apps/src/templates/teacherDashboard/CourseCard.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import color from "../../util/color";
+import ReactTooltip from 'react-tooltip';
+import FontAwesome from '../FontAwesome';
+import i18n from "@cdo/locale";
+import _ from 'lodash';
+
+const styles = {
+  card: {
+    overflow: 'hidden',
+    borderRadius: 2,
+    border: '1px solid gray',
+    position: 'relative',
+    height: 200,
+    width: 400
+  },
+  overlay: {
+    background: 'linear-gradient(to right, rgba(2,130,132,.95), rgba(2,130,132,0))',
+    bottom: 0,
+    left: 0,
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    zIndex: 1,
+  },
+  image: {
+    position: 'absolute',
+  },
+  courseName: {
+    paddingLeft: 20,
+    paddingRight: 10,
+    paddingTop: 10,
+    paddingBottom: 5,
+    marginTop: 15,
+    fontSize: 18,
+    fontFamily: '"Gotham 3r", sans-serif',
+    color: 'rgba(255, 255, 255, .9)',
+    zIndex: 2,
+    position: 'absolute',
+    display: 'inline'
+  },
+  description: {
+    paddingLeft: 20,
+    paddingRight: 10,
+    paddingTop: 10,
+    paddingBottom: 5,
+    marginTop: 90,
+    fontSize: 14,
+    fontFamily: '"Gotham", sans-serif',
+    color: color.gray,
+    background: color.white,
+    height: 100,
+    position: 'absolute',
+    zIndex: 2,
+  },
+  checkIcon: {
+    position: 'absolute',
+    zIndex: 3,
+    fontSize: 18,
+    color: color.white,
+    marginLeft: 350,
+    background: color.teal,
+    padding: 10,
+    borderRadius: 100,
+    border: '1px solid white',
+    display: 'inline',
+    marginTop: 10
+  },
+  continueLink: {
+    color: color.teal,
+    fontSize: 12,
+    fontFamily: '"Gotham", sans-serif',
+    fontWeight: 'bold',
+    marginTop: -2,
+    display: 'inline'
+  },
+  chevron: {
+    display: 'inline',
+    color: color.teal,
+    fontSize: 10,
+    fontWeight: 'bold',
+    marginLeft: 5,
+  },
+  linkBox: {
+    display: 'block',
+    paddingBottom: 10
+  },
+};
+
+const CourseCard = React.createClass({
+  propTypes: {
+    cardData: React.PropTypes.shape({
+      courseName: React.PropTypes.string.isRequired,
+      description: React.PropTypes.string.isRequired,
+      image: React.PropTypes.string.isRequired,
+      link: React.PropTypes.string.isRequired,
+      assignedSections: React.PropTypes.array.isRequired
+    })
+  },
+
+  checkEnrollment() {
```

Comment:
Is it worth adding a unit test for this business logic?

Also the name implies it would return the result of a check, which is typically a boolean or one of several vaules, but it actually returns a span DOM React element?  Perhaps the name should indicate that more clearly?